apps/balancer-tools/src/app/apr/layout.tsx

```diff
@@ -1,24 +1,28 @@
-"use client";
-
-import { useParams } from "next/navigation";
+import { Metadata } from "next";
 import * as React from "react";
-import invariant from "tiny-invariant";
 
 import { Header } from "#/components/Header";
 
 import HeaderEndButton from "./(components)/HeaderEndButton";
 
-export default function Layout({ children }: React.PropsWithChildren) {
-  const { roundId, poolId, network } = useParams();
-  invariant(!Array.isArray(roundId), "roundId cannot be a list");
-  invariant(!Array.isArray(poolId), "poolId cannot be a list");
-  invariant(!Array.isArray(network), "network cannot be a list");
+export const metadata: Metadata = {
+  title: "Balancer - APR Enhancement Calculator",
+  description: "APR Enhancement Calculator for Balancer pools through rounds",
+};
 
+export default function Layout({
+  children,
+  params,
+}: {
+  children: React.ReactNode;
+  params: { roundId: string; poolId: string; network: string };
+}) {
+  const { roundId, poolId, network } = params;
   return (
     <div className="flex h-full flex-col">
       <Header
         linkUrl={"/apr"}
-        title={"Pool Simulator"}
+        title={"APR Enhancement Calculator"}
```

Comment:
Here too

Reply:
Would create a constant with the name and use it in both places